internal/controller/tempo/tempostack_controller.go

```diff
@@ -201,6 +202,7 @@ func (r *TempoStackReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	}
 
 	builder := ctrl.NewControllerManagedBy(mgr).
+		Named("tempostack-"+uuid.New().String()).
```

Comment:
Is a uuid required here?
I think adding the CR name also works, like in this commit for the kubebuilder repo: https://github.com/kubernetes-sigs/kubebuilder/commit/79a06e54a9e6a6988ab23f419a0079f9aa61261f

Reply:
perhaps yes :D

Reply:
Removed, you're right, we don't need it.